api/models/model.py

```diff
@@ -74,6 +74,7 @@ class App(db.Model):
     is_public = db.Column(db.Boolean, nullable=False, server_default=db.text('false'))
     is_universal = db.Column(db.Boolean, nullable=False, server_default=db.text('false'))
     tracing = db.Column(db.Text, nullable=True)
+    max_active_requests = db.Column(db.Integer, nullable=False, server_default=db.text('0'))
```

Comment:
Is `max_active_requests` reserved? I don't see any configuration for the front end interface.

Reply:
I directly modify the max_active_requests in the database to customize the rate limiting configuration for a specific application. Because my frontend skills are not very strong, there is currently no frontend interface for this. Perhaps in the future, other community members could add an editing feature for this field in the frontend assistant's modification page.